docs/pages/deployment/monitoring.rst

```diff
@@ -7,7 +7,7 @@ Health checks
 *************
 
 An endpoint is provided to perform health checks on the Nuts node.
-It reports in a format compatible with :ref:`Spring Boot's Health Actuator <https://docs.spring.io/spring-boot/docs/2.0.x/actuator-api/html/#health>`.
+It reports in a format compatible with `Spring Boot's Health Actuator <https://docs.spring.io/spring-boot/docs/2.0.x/actuator-api/html/#health>`__.
```

Comment:
Single underscore at end

Reply:
Only if you want to refer to them see: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks and https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use